sys/net/include/etx_beaconing.h

```diff
@@ -21,27 +21,30 @@
 #define ETX_BEACONING_H_
 #include <stdint.h>
 #include "sixlowpan.h"
```

Comment:
Add an `@file` block and `@defgroup`, `@ingroup` etc. Look at other header files nearby for examples.